src/cr/cube/matrix/measure.py

```diff
@@ -613,18 +611,6 @@ def _intersections(self):
             "%s must implement `._intersections`" % type(self).__name__
         )
 
-    def _smoother(self, smoothing_spec):
-        """SingleSidedMovingAvgSmoother object used for smoothed measures.
-
-        Raises `NotImplementedError` if the function in the smooting spec is different
-        from `one_sided_moving_avg` the only function available today as smoothing
-        algorithm.
-        """
-        function = smoothing_spec.function
-        if function != "one_sided_moving_avg":
-            raise NotImplementedError("Function {} is not available.".format(function))
-        return SingleSidedMovingAvgSmoother(smoothing_spec.window, self._dimensions[-1])
-
```

Comment:
This goes away cause the actual smoother is returned directly in the `SmoothingSpec` factory